index.html

```diff
@@ -34,5 +33,181 @@ <h1 class="header" >I’m Mariane<br>
+                </article>
+                    <a class="nav-button" href="#"><input type="button" class="mybutton" value="See Project"></a> 
+        </section>
+
+        <section class="border_section3">
+                <div class="image_section3">
+                    <img src="image/art_printing.svg" alt="Professional Art Printing">
+                </div>
+                <header class="border_heading">
+                    <h2 class="heading">Multi-Post<br>
+                        Stories</h2>
+                    <div class="flex-container">
+                        <ul>
+                        <li class="blue">CANOPY</li>
+                        <li><img src="image/Counter.svg" alt="Counter"></li>
+                        <li class="blue">Back End Dev</li>
+                        <li><img src="image/Counter.svg" alt="Counter"></li>
+                        <li class="blue">2015</li>
+                        </ul>
+                    </div>
+                </header>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <div class="tags">
+                    <ul>
+                    <li class="tag1"><img src="image/Tag_html.svg" alt="HTML Tag"></li>
+                    <li class="tag2"><img src="image/Tag_css.svg" alt="HTML Tag"></li>
+                    <li class="tag1"><img src="image/Tag.svg" alt="HTML Tag"></li>
+                    </ul>
+                </div>
+            </article>
+                    <a class="nav-button" href=""><input type="button" class="mybutton" value="See Project"></a> 
+        </section>
+        </div>
+
+        <section class="container_resume" >
+            <article class="resume_border">
+                <h2 class="heading_resume">About<br>Myself</h2>    
+            <p class="content_resume">Hello I’m a software developer! I can help you build a product , feature or website
+                     Look through some of my work and experience! If you like what you see and have a 
+                     project you need coded, don’t hestiate to contact me.</p>
+                     <a class="connect_resume" href="#"><img src="image/Social Media.svg" alt="Let's Connect"></a>
+                     <div class="social_resume">
+                        <ul>
+                            <li><a href="#"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                            <li><a href="#"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                            <li><a href="#"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                            <li><a href="#"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                            <li><a href="#"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
```

Comment:
-[Optional] Kindly, it is better to add your social media link to `href` to navigate to your social media with icons,💕❤️